crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -4,23 +4,12 @@
 use super::AudioBackend;
 use std::{thread};
 use std::{cell::Cell, rc::Rc};
-use crate::{Error, Result};
+use crate::{Result};
```

Comment:
```suggestion
use crate::Result;
```
I would recommend running the codebase through rustfmt `cargo fmt --all`